### ISSUE TITLE
fedy fc22

### ISSUE THREAD

**Issue by alvesadrian** [folkswithhats/fedy#271]
fedy is not working on FC22

**Comment by DStewart1997** [comment]
What do you mean it isn't working? It won't install? It won't open? It won't install programs?

**Comment by gregharvey** [comment]
Hi, actually Adrian was asking on my behalf and actually, it seems fine. Multiple things happened to give the impression there was no F22 release:

1. fedup looked for http://download.opensuse.org/repositories/home:/satya164:/fedy/Fedora_$releasever/ which is fine for F21, but no longer the location of the repo for F22. So I assumed there simply hadn't been an update yet; but...

2. I also went to download fedy-installer, just this afternoon actually, and for some reason the repo installer in my apparently downloaded copy was still using the opensuse.org repo - has that only just been changed or was it somehow cached? I don't know.

Anyway, I just downloaded fedy-installer *again*, right now, to double check and I see the new F22 repo is there in the installer:

http://folkswithhats.org/repo/$releasever/

Which now works fine! Just upgraded Fedy, no issue.

So sorry about the noise, it was a mix of `fedup` not knowing the new repo location and the (weird) inability to pull down the latest fedy-installer script to check the current location.

Thanks!
